### DISCUSSION TITLE
[QUESTION] How to delete longhorn-system namespace

### DISCUSSION THREAD

**Discussion by themonaminor** [longhorn/longhorn#3742] [General]
longhorn-system namespace is not deleted, it keep in terminating state.

 kubectl delete ns longhorn-system
namespace "longhorn-system" deleted

longhorn-system     Terminating   25m

**Comment by joshimoo** [comment]
Have a look at this part of the documentation: https://longhorn.io/docs/1.2.3/deploy/uninstall/
